apps/backend/repository/repository.controller.js

```diff
@@ -78,10 +78,17 @@ const includeRepositoryTags = (query) => {
 };
 
 async function index({ query, user, opts }, res) {
-  const { search, name, schemas } = query;
+  const { search, name } = query;
+  const availableSchemas = (process.env.NUXT_PUBLIC_AVAILABLE_SCHEMAS || '')
```

Comment:
At some point we should consider getting the config via API to avoid duplication and need for special Docker image build